client/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: 'standard-with-typescript',
+  overrides: [
+    {
+      env: {
+        node: true
+      },
+      files: [
+        '.eslintrc.{js,cjs}',
+        '*.ts'
+      ],
+      parserOptions: {
+        sourceType: 'script'
+      }
+    }
+  ],
+  parserOptions: {
+    ecmaVersion: 'latest',
```

Comment:
do we need to keep this the same as the `target` in `tsconfig.json`?

Reply:
Don't know. These are the default options when I ran `npm init @eslint/config`. Indeed, everything seems to work just the same when I delete it.

https://eslint.org/docs/latest/use/configure/language-options#specifying-parser-options